### PR TITLE
Introduce logging to log activities over kernel permissions

### PR DESCRIPTION
**Pull request by ayushjn20**

### CODE REVIEW

File: omniport/core/kernel/utils/logs.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import logging
+
+logger = logging.getLogger('kernel_permissions')
+
+
+def log_permission(permission_name, user, is_allowed):
+    """
+    A common utility function to log requests for kernel permissions
```
Comment (dhruvkb) [inline omniport/core/kernel/utils/logs.py]: Function documentation should be imperative like
- Do X
- Change Y
- Return Z

File: omniport/omniport/settings/configuration/logging.py [hunk 2]
```diff
@@ -72,6 +82,16 @@
             ],
             'propagate': False,
         },
+        'django.server': {
```
Comment (dhruvkb) [inline omniport/omniport/settings/configuration/logging.py]: Isn't `django.server` already covered under `django`? There shouldn't be a need to define it again separately.

File: omniport/core/kernel/utils/logs.py (hunk 1 above)
Comment (ayushjn20) [reply omniport/core/kernel/utils/logs.py]: Done, please check.

File: omniport/omniport/settings/configuration/logging.py (hunk 2 above)
Comment (ayushjn20) [reply omniport/omniport/settings/configuration/logging.py]: Missed that, my mistake. Check again

**Comment by ayushjn20** [comment]
> One change and one question but otherwise pretty awesome!
> 
> This one's blame falls on me but it does seem to me that there is immense repetition across the files in the `permissions/` directory? Could you put in some effort and reduce this repetition?

I can try, but I think the purpose of this PR has been fulfilled. If I am able to reduce it, I will create a separate PR.

File: omniport/core/kernel/utils/logs.py (hunk 1 above)
Comment (dhruvkb) [reply omniport/core/kernel/utils/logs.py]: @ayushjn20 I don't see the difference.
```suggestion
    Form sentences to log requests for kernel permissions
```

**Pull request merged**